softlayer/cpi-dynamic.yml

```diff
@@ -22,16 +22,15 @@
   path: /resource_pools/name=vms/cloud_properties?
   value:
     domain: ((sl_vm_domain))
-    vmNamePrefix: ((sl_vm_name_prefix))
-    ephemeralDiskSize: 100
-    startCpus: 4
-    maxMemory: 8192
-    deployedByBoshcli: true
-    datacenter:
-      name: ((sl_datacenter))
-    hourlyBillingFlag: true
-    networkComponents:
-    - maxSpeed: 100
+    hostname_prefix: ((sl_vm_name_prefix))
+    ephemeral_disk_size: 100
+    cpu: 4
+    memory: 8192
+    deployed_by_boshcli: true
```

Comment:
What does this property do?

Reply:
For supporting deploying director with dynamic IP, we have to use domain and map the private IP to the domain in the /etc/hosts, but it's not needed for cf deployment. However, this design can't work well now since it's a must to set certs for director components like nats. I opened the [issue 282](https://github.com/cloudfoundry/bosh-softlayer-cpi-release/issues/282) to CPI to remove this part of code, including the property ` deployed_by_boshcli`. We will update bosh-deployment repo again once we solve that CPI issue.